packages/agreement-platformstate-writer/src/index.ts

```diff
@@ -13,7 +13,7 @@ import { handleMessageV1 } from "./consumerServiceV1.js";
 import { handleMessageV2 } from "./consumerServiceV2.js";
 import { config } from "./config/config.js";
 
-const dynamoDBClient = new DynamoDBClient({});
+const dynamoDBClient = new DynamoDBClient();
```

Comment:
Double check. It should be the same. The constructor was already used without `{}` in other parts of the code like `catalog-platformstate-writer`, `purpose-platformstate-writer` and the `bff`.